script/openqa-bootstrap

```diff
@@ -59,7 +59,7 @@ fi
 
 if ping -c1 gitlab.suse.de. ; then
 	# clone SLE needles if run from within SUSE network
-	[ -d /var/lib/openqa/tests/opensuse/products/sle/needles ] || git clone https://gitlab.suse.de/openqa/os-autoinst-needles-sles.git /var/lib/openqa/tests/opensuse/products/sle/needles
+	[ -d /var/lib/openqa/tests/opensuse/products/sle/needles ] || git clone --depth 1 https://gitlab.suse.de/openqa/os-autoinst-needles-sles.git /var/lib/openqa/tests/opensuse/products/sle/needles
```

Comment:
>@bzoltan1 Very valid point. I will be happy to work on the UX of this tools step by step. But I personally prefer breaking down stories to simple PRs.

Maybe I expressed incorrectly. It would be good to place similar message like in fetchneedles into this place. Otherwise you are silently doing  action which is is not user aware about, but user should be noticed. Change in the behavior and notice about it have to be in one PR in this case.

Reply:
Makes sense. Done.